doozer/doozerlib/release_inspector.py

```
@@ -65,19 +65,22 @@ async def introspect_release(pullspec: str) -> Model:
     return release_info
 
 
-async def extract_nvr_from_pullspec(pullspec: str, arch: str = None) -> Tuple[str, str, str]:
+async def extract_nvr_from_pullspec(
+    pullspec: str, arch: str = None, registry_config: str = None
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)